backend/src/schema/types/Todo.js

```diff
@@ -4,7 +4,7 @@ export const TodoType = gql`
   type Todo {
     id: ID
     text: String
-    author: Author
+    author: User @relation(name: "ASSIGNED", direction: "IN")
```

Comment:
You're lacking a type resolver for `Todo.author`. If you return a type `Todo` in a mutation which is not implemented with `neo4j-graphql-js` then `author` will be `undefined` and your server will crash. 

Here's what I mean
```js
export default {
  Query: {/* ... */},
  Mutation: {/* ... */},
  Todo: {
    author: () => {} // make sure this resolves the author for each todo
  },
}
```
With `neo4j-graphql-js` you are just lucky that all the data can be gathered in one big database call, including all related objects like `author`. But if you have a mutation like:
```gql
  mutation addToDo($text: String!, $authorName: String!) {
    addToDo(text: $text, authorName: $authorName) {
      id
      text
      author {
        name
      }
    }
  }
```
and this one goes to the database then your server might respond `undefined` for `author` and it will crash. Apparently, your tests still work because you did not refactor the mutations to save state to the database.

So: Implement type resolvers and admire the beauty of graphql.

Reply:
You and I talked about this couple weeks ago. `neo4j-graphql-js` generates a bunch of resolvers for me that fetch the data from the neo4j DB. Since I didn't implement the resolver myself it uses the autogenerated one.
So no issue here. It is intentional.

In HC you disabled a lot of the resolver generation, so there you have to implement it.

Reply:
OK! Fair enough. If you only use `neo4j-graphql-js` then you don't have an issue because it's fetching all the related objects in a single database call (based on `resolveInfo` as input).